hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.tsx

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
```

Comment:
Weren’t we planning to add a button or icon that allows a column to be removed from the table when clicked? These icons should appear when we include a column in the table. Each icon has an 'X' mark, and clicking on it will remove the corresponding column from the view, as well as the icon itself? Did we plan on removing it?

Reply:
Yes, currently we found that adding the tags is adding an extra step to a simple task, hence for the time being we have stuck with just using the dropdown to select/de-select until we find a better way to implement this selection which is more intuitive